packages/loader/src/ShaderChunkLoader.ts

```
@@ -0,0 +1,45 @@
+  Loader,
+  ResourceManager,
+  ShaderFactory,
+  resourceLoader,
+  // @ts-ignore
+  ShaderLib
+} from "@galacean/engine-core";
+import { Utils } from "./Utils";
+
+@resourceLoader("ShaderChunk", ["glsl"])
+class ShaderChunkLoader extends Loader<void[]> {
+  load(item: LoadItem, resourceManager: ResourceManager): AssetPromise<void[]> {
```

Biome

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)